lib/api.js

```diff
@@ -518,6 +518,8 @@ var parseDoc = function(json) {
     json.href,
     json.tags,
     slugs,
+    jsogit.first_publication_date,
+    json.last_publication_date,
```

Comment:
Can we wrap these with a `new Date(..)` to actually cast them to a date? Else they are a string and everyone in user land needs to convert them.